src/App.vue

```diff
@@ -438,6 +442,13 @@ export default class App extends Vue {
     return +sessionStorage.getItem('IA_SURVEY_ID')
   }
 
+  /** Get banner text. */
+  get bannerText (): string | null {
```

Comment:
I don't think you need to declare the result might be null (since all types can be null).

And actually, line 449 might return undefined (but I don't think you need to declare that either), though it might be a good idea to add `|| null` to this line.

Reply:
Fixed.